dags/hacker_news.py

```diff
@@ -76,18 +63,66 @@ def process_stories_ids(date: str, **context) -> str:
         return stories_info
 
     @task()
-    def save_data():
+    def save_data(**context):
         """
         Save processed data to storage server
         """
+        import io
+
+        from minio import Minio     # type: ignore
+
+        global FILENAME
+        global BUCKET_PATH
+        global BUCKET_NAME
```

Comment:
Why we need `global` here?

Reply:
Interesting! Didn't hear about such technique, is there any reference or PEP about it?

Reply:
In fact, it's even more vkusovschina, I looked on PEP, it doesn't say anything about it

Reply:
Why not to include all the global names like imported modules and packages then?